### PR TITLE
20230707(Johnny)

### PR DESCRIPTION
**Pull request by JohnnyHuang1104**
Topic: Make partial menu item can switch to their animation
Content:
1.ViewPager - 
(1)Add basic animation to regular item.
(2)Add preview animation to  one_screen_three_page_basic item.
(3)Add preview and zoom animation to one_screen_three_page_advanced item.
2.PageIndicator - 
(1)Add PageIndicator to page_indicator item,but it can't turn off now.

### CODE REVIEW

File: app/src/main/java/com/example/javaviewpager2/MainActivity.java [hunk 1]
```diff
@@ -38,23 +39,70 @@ protected void onCreate(Bundle savedInstanceState) {
...
+
+    // 將menu.xml的Layout與Item匯入activity_main.xml
+    @SuppressLint("ResourceType")
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.menu, menu);
+        return true;
+    }
+
+    // 獲得menu上不同Item的Id，並將對應的效果放到activity_main.xml。
```
Comment (Wujuichen) [inline app/src/main/java/com/example/javaviewpager2/MainActivity.java:59]: 註解的很詳細。

File: app/src/main/java/com/example/javaviewpager2/MainActivity.java [hunk 2]
```diff
@@ -38,23 +39,70 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    public boolean onOptionsItemSelected(MenuItem item) {
+        int id = item.getItemId();
```
Comment (Wujuichen) [inline app/src/main/java/com/example/javaviewpager2/MainActivity.java:61]: 註解的很詳細

File: app/src/main/res/layout/page.xml [hunk 3]
```diff
@@ -100,15 +100,18 @@
         android:layout_height="50dp"
         android:layout_alignParentLeft="true"
         android:padding="8dp"
-        android:src="@drawable/left" />
+        android:background="@drawable/left"
+
+        />
```
Comment (Wujuichen) [inline app/src/main/res/layout/page.xml]: 這是不小心空一格?

File: app/src/main/java/com/example/javaviewpager2/MainActivity.java [hunk 4]
```diff
@@ -69,11 +117,11 @@ private List<MyPerson> getPersonList() {
         return myList;
     }
 
-    private void setViewPagerScroll() {
+    private void setViewPagerScroll(Boolean clip) {
```
Comment (Wujuichen) [inline app/src/main/java/com/example/javaviewpager2/MainActivity.java:120]: 基本上沒什麼大問題

File: app/src/main/res/layout/page.xml (hunk 3 above)
Comment (JohnnyHuang1104) [reply app/src/main/res/layout/page.xml]: 對XDD

**Pull request merged**